config/environment.rb

```diff
@@ -1,3 +1,19 @@
+# Strip whitespace from environment values for proper API to app handshake.
+class AppEnv
+  def self.[](key)
+    ENV.key?(key) && ENV[key] = ENV[key].strip
```

Comment:
For this part `ENV.key?(key)` are we checking that the key is not null? Also, why would we want to include `&&` instead of just separating these onto different lines?

Reply:
Yes, it is a check that a key is not null (kind of functions as a defensive check at this point). The && is included so that if `ENV.key?(key)` is equal to null, the second part of the statement, `ENV[key] = ENV[key].strip` will not be evaluated. If this is not the proper way to do this, let me know.

Reply:
This code snippet is slightly outdated though - `ENV[key] = ENV[key].strip` is actually just `ENV[key].strip` so that the stripped whitespace ENV value is returned from the method call rather than mutating ENV permanently.